compiler/src/test/java/com/dylibso/chicory/approvals/ApprovalTest.java

```diff
@@ -69,7 +70,7 @@ public void verifyI32() {
     public void verifyI32Renamed() {
         var module = parse(getSystemClassLoader().getResourceAsStream("compiled/i32.wat.wasm"));
         var result = Compiler.builder(module).withClassName("FOO").build().compile();
-        verifyClass(result.classBytes(), false);
+        verifyClass(result.classBytes(), (name) -> false);
     }
```

Comment:
We have finer grained control of which classes are included in the approval now.  what is it that `verifyI32Renamed` is focusing on?  Is it just the Shaded class?

Reply:
it's focusing on emitting on the functionality of `withClassName`, any better/alternative proposal for testing this bit is welcome, there is space for improvement here.